application/classes/beans/vendor.php

```diff
@@ -1054,7 +1054,7 @@ protected function _return_form_lines_array($lines)
 	@attribute account OBJECT The #Beans_Account# to categorize the cost.
 	@attribute description STRING 
 	@attribute amount DECIMAL
-	@attribute quantity INTEGER Must be >= 0
+	@attribute quantity DECIMAL Must be >= 0
```

Comment:
And let's update these to be the same as the customer-side ones...  that's lagging documentation that should have been fixed a while ago.

```
@required @attribute lines quantity DECIMAL The number of units ( up to three decimal places ).
```